hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -825,18 +834,16 @@ private String getSSTFullPath(String sstFilenameWithoutExtension,
    *         e.g. ["/path/to/sstBackupDir/000050.sst",
    *               "/path/to/sstBackupDir/000060.sst"]
    */
-  public synchronized Optional<List<String>> getSSTDiffListWithFullPath(
-      DifferSnapshotInfo src,
-      DifferSnapshotInfo dest,
-      String sstFilesDirForSnapDiffJob
-  ) throws IOException {
+  public synchronized Optional<List<String>> getSSTDiffListWithFullPath(DifferSnapshotInfo src,
+                                                                        DifferSnapshotInfo dest,
+                                                                        String sstFilesDirForSnapDiffJob) {
```

Comment:
nit: Indenting parameters like this triggers unnecessary change whenever visibility / return type / method name is updated.